configure.ac

```
@@ -227,6 +229,18 @@ AM_COND_IF([ENABLE_MANPAGES], [
 	AS_IF([test "x$RST2MAN" = "x"], [AC_MSG_ERROR([[--enable-manpages requires rst2man]])])
 ])
 
+AC_ARG_WITH([bash-completion],
+	AS_HELP_STRING([--with-bash-completion@<:@=DIR@:>@], [install bash completion script in DIR (or the bash completions default directory if not specified)]),
+	[bashcompdir="${withval}"],
+	[]
+)
+AM_CONDITIONAL([WITH_BASH_COMPLETION], [test "x$with_bash_completion" != "x"])
+AS_IF([test "x$with_bash_completion" = "xyes"], [
+   PKG_CHECK_MODULES([BASH_COMPLETION], [bash-completion >= 2.0],
+	[bashcompdir=$($PKG_CONFIG --variable=completionsdir bash-completion)],
+	[bashcompdir="$datadir/bash-completion/completions"])
+])
```

Make the option definition more compact and fix for `--without-bash-completion`.

```suggestion
	AS_HELP_STRING([--with-bash-completion@<:@=DIR@:>@], [install bash completion
		script in DIR (or the bash completions default directory if not specified)]))
AM_CONDITIONAL([WITH_BASH_COMPLETION], [
	test "x$with_bash_completion" != "x" -a "x$with_bash_completion" != "xno"])
AS_IF([test "x$with_bash_completion" = "xyes"], [
	PKG_CHECK_MODULES([BASH_COMPLETION], [bash-completion >= 2.0],
		[bashcompdir=$($PKG_CONFIG --variable=completionsdir bash-completion)],
		[bashcompdir="$datadir/bash-completion/completions"])
	], [bashcompdir="$with_bash_completion"])
```